src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -2066,7 +2078,8 @@ export default class Aircraft {
                 }
 
                 this.target.speed = crange(3, offset[1], 10, this.model.speed.landing, this.fms.currentWaypoint().start_speed);
-            } else if ((this.altitude - runway_elevation) >= 300 && this.mode === FLIGHT_MODES.LANDING) {
+            }
```

Comment:
It is a little scary, that's normal.  I've been there.  I can remember going over a simple thing with my mentor a while back, and we went down this same path where a 100(ish) line block of code became a whole bunch of files.  I asked, 'How the hell is this better!?'  

You'll see.  I might just do these changes as a PR so we can talk about it.  I fully expect conversations about the whys and the 'I dont get what this is doing', or just plain 'Whaaaaat??'.  Totally normal, been there. 

I've said it before, but just to reiterate, _please_ ask questions.  You're doing a wonderful job of that so far.  Continue that.  Ask when you see something you don't understand or when you have questions.  That's how we all get better as developers. 

Hmm, I am both encouraged and terrified by that hashtag...